src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -142,4 +163,18 @@ class CosmasGoogleCloudService(private val bucketName: String,
             LOG.info("Deleting file failed: ${e.message}")
         }
     }
+
+    fun getPatchList(projectId: String, fileId: String): MutableList<CosmasProto.FileInformation.Patch>? {
```

Comment:
Normally you'll want to return immutable collections, unless you really want to tell the caller that it is okay to modify the collection directly (it is not the case)

Reply:
ok